pkg/cmd/deadman/deadman.go

```diff
@@ -0,0 +1,93 @@
+package deadman
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	configv1informers "github.com/openshift/client-go/config/informers/externalversions"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"k8s.io/component-base/version"
+
+	configeversionedclient "github.com/openshift/client-go/config/clientset/versioned"
+	"github.com/openshift/library-go/pkg/controller/controllercmd"
+)
+
+type Options struct {
+	controllerContext *controllercmd.ControllerContext
+
+	challengeInterval             time.Duration
+	controllerResponseGracePeriod time.Duration
+}
+
+func NewDeadmanCommand(ctx context.Context) *cobra.Command {
```

Comment:
nit: can you please add some godoc here explaining why we added this and some example cases when then helps? (for future generations)